welleng/errors/tool_errors.py

```diff
@@ -1,12 +1,11 @@
+import os
+
 import numpy as np
-from numpy import sin, cos, tan, pi, sqrt
-from numpy.core.defchararray import index
 import yaml
-import os
+
 from collections import OrderedDict
-# import imp
+from numpy import sin, cos, tan, pi, sqrt
 
-# import welleng.error
```

Comment:
- Sorted the imports
- Removed `from numpy.core.defchararray import index` since it wasn't used in the file.